### DISCUSSION TITLE
Standup (Monday, October 24th)

### DISCUSSION THREAD

**Discussion by antonieto** [BBGTC/food-bank-app#20] [General]
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Opened #18 

**What are your priorities for the next 24 hours?**:
 - Setup Github actions for the Expo app

**Blockers**:
 - Semana TEC

**Shoutouts**:
 - @DavBot02 for his offer at Google

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Nothing (baquetón)

**What are your priorities for the next 24 hours?**:
 - Work on styles/navigation

**Blockers**:
 - Baquetón

**Shoutouts**:
 - @antonieto for his PR

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Relaxed, very well needed

**What are your priorities for the next 24 hours?**:
 - Take #9

**Blockers**:
 - None

**Shoutouts**:
 - @DavBot02  for his offer @google

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  React  review

**What are your priorities for the next 24 hours?**:
 -  Check github actions with @antonieto 

**Blockers**:
 - lack of will

**Shoutouts**:
 - @DavBot02  for his offer at Google :)
